src/components/title/title.js

```diff
@@ -0,0 +1,9 @@
+import s from './title.scss';
+
+const Title = ({ title }) => (<h1 className={s.primary}>{title}</h1>);
```

Comment:
`const Title = ({ children }) => (<h1 className={s.primary}>{children}</h1>);`

and for children we don't need to specify type